### PR TITLE
Update ar.jsx

### PR DESCRIPTION
**Pull request by ghost**
Adding more missing strings. Some others to come in few minutes. Just saving this one.

### CODE REVIEW

File: app/assets/javascripts/components/locales/ar.jsx [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
   "account.block": "حظر @{name}",
+  "account.disclaimer": "هذا المستخدم من مثيل خادم آخر. قد يكون هذا الرقم أكبر.",
+  "account.edit_profile": "تعديل الملف الشخصي",
   "account.follow": "إتبع",
-  "account.posts": "المشاركات",
-  "account.follows": "يتبع",
   "account.followers": "المتابعون",
   "account.follows_you": "يتابعك",
+  "account.follows": "يتبع",
+  "account.mention": "أُذكُر @{name}"
```
Comment (yookoala) [inline app/assets/javascripts/components/locales/ar.jsx]: This line (line 17) missed a comma "," at the end.

**Comment by yookoala** [comment]
Recommend you to use better editor to do the translation.

For example, [Atom.io](https://atom.io/) with [linter-eslint](https://atom.io/packages/linter-eslint) and [linter-js-yaml](https://atom.io/packages/linter-js-yaml) can catch most problem you have in your syntax.

**Review by yookoala — CHANGES_REQUESTED** [review]
Please fix the syntax problem so it can pass Travis CI tests.

**Comment by ghost** [comment]
Oh ! Sorry ! I'm going to fix this right now.

File: app/assets/javascripts/components/locales/ar.jsx (hunk 1 above)
Comment (ghost) [reply app/assets/javascripts/components/locales/ar.jsx]: Corrected. Sorry again. I'll double check next time.

**Review by yookoala — APPROVED** [review]

**Comment by yookoala** [comment]
No worry. Human can easily make mistake. That's why we need tools.

**Pull request merged**